syft/frameworks/torch/hook/hook.py

```diff
@@ -177,6 +180,12 @@ def __init__(
         syft.local_worker = self.local_worker
         syft.hook = self
 
+    def create_wrapper(cls, child_to_wrap):
```

Comment:
This does not interact with the child

Reply:
For the TF side, we will need to create `tf.Tensor` and `tf.Variable` wrappers depending on what chain we're wrapping, so this method needs that parameter, it just happens to be unused for torch.

Reply:
Ah okay. Could you add a comment about that here so people know this isn't a bug